README.md

```diff
@@ -1 +1,583 @@
+
+## Testing the services
+
+### Setting up the environment  
+
+Please refer to the [Prerequisites](#prerequisites) section for information on setting up the required tools and environment. Access the below URLs and make sure you have Prometheus, Grafana and Jaeger servers up and running.
+- Prometheus - [http://localhost:19090](http://localhost:19090)
+- Grafana - [http://localhost:3000](http://localhost:3000)
+- Jaeger - [http://localhost:16686](http://localhost:16686)
+
+Create the database using the script provided in resources directory. This datbase is being used by the services to retrieve data.
+```sql
+mysql -u <username> -p
+source resources/mysql.sql;
+```
+
+### Invoking the service
+
+- Navigate to `microservices-observability/guide` and run the following commands in separate terminals to start all four HTTP services. This will start the `Airline Reservation`, `Hotel Reservation`, `Car Rental` and `Travel Agency` services in ports 9091, 9092, 9093 and 9090 respectively.
+
+```bash
+   $ ballerina run travel_agency/
+```
+```bash 
+   $ ballerina run -e b7a.observability.metrics.prometheus.port=9797 airline_reservation/
+```
+```bash
+   $ ballerina run -e b7a.observability.metrics.prometheus.port=9798 hotel_reservation/
+```
+```bash
+   $ ballerina run -e b7a.observability.metrics.prometheus.port=9799 car_rental/
+```
+Note that we are overridng the prometheus port by passing the parameter (`-e b7a.observability.metrics.prometheus.port=xxxx`) for all the services, except the travel_agency service. Travel agency service will publish the metrics data on the default port defined in the `ballerina.conf` file (9796). This will avoid any conflicts on port being already occupied.
+- Invoke the Travel Agency service by sending a POST request to arrange a tour.
+
+```bash
+   curl -v -X POST -d \
+   '{"ArrivalDate":"2007-11-06", "DepartureDate":"2007-11-06", "From":"CMB", "To":"DXB", 
+   "VehicleType":"Car", "Location":"Changi"}' \
+   "http://localhost:9090/travel/arrangeTour" -H "Content-Type:application/json" 
+```
+
+- Travel Agency service will send a response similar to the following:
+    
+```bash
+   HTTP/1.1 200 OK
+    {
+        "Flight":{
+            "flightNo":1,"airline":"Emirates","arrivalDate":"2007-11-06+05:30",
+            "departureDate":"2007-11-06+05:30","to":"DXB","rom":"CMB","price":100
+        },
+        "Hotel":{
+            "HotelName":"Elizabeth","FromDate":"2007-11-06","ToDate":"2007-11-06","DistanceToLocation":2
+        },
+        "Vehicle":{
+            "company":"Sixt","arrivalDate":"2007-11-06+05:30","departureDate":"2007-11-06+05:30",
+            "vehicleType":"Car","price":30
+        }
+    }
+```
+## Observability 
+Ballerina is by default observable. Meaning, you can easily observe your services, resources, etc. Observability configurations are defined in the `ballerina.conf` file. A sample configuration file can be found in `microservices-observability/guide/` directory.
```

Comment:
Shall we rephrase this to something like the one in https://github.com/ballerina-guides/restful-service/blob/master/README.md#observability ?